app/src/main/java/waiter/ClientConnection.java

```diff
@@ -8,20 +8,20 @@
 
 public class ClientConnection implements Connection {
 
-    private final Socket client;
+    private final Socket clientSocket;
 
-    ClientConnection(Socket client) {
-        this.client = client;
+    ClientConnection(Socket clientSocket) {
```

Comment:
I think this is another mismatch, In the `Listener` class we have `serverSocket` but here we're calling this `clientSocket` I think they mean the same thing in terms of socket. I would suggest either just go with the name server socket or just socket since I don't think clientSocket is a thing because sockets in general is a server term. I also noticed that we're using `import java.net.Socket` here but in `ClientConnection` we're using `import java.net.ServerSocket`. What are the difference between the two and can we just one instead of both?

Reply:
I fixed the `clientSocket` to just be `socket` in [this commit](https://github.com/jonmanahan/waiter/pull/2/commits/c5b65f8f2b576acc81c13d122850ed83f4052bda). In the case of the ServerSocket, we use that in listener instead of a regular socket as it has the additional feature of the `accept` method, which we need to get the socket in the `ClientConnection` above, if that makes sense.

Reply:
Make sense. Thanks for clarifying!